sentinel-core/src/main/java/com/alibaba/csp/sentinel/log/jul/ConsoleHandler.java

```diff
@@ -42,6 +47,9 @@ class ConsoleHandler extends Handler {
      * A Handler which publishes log records to System.err.
      */
     private StreamHandler stderrHandler;
+    @SuppressWarnings("PMD.ThreadPoolCreationRule")
+    private final ExecutorService executor = Executors.newSingleThreadExecutor(
+            new NamedThreadFactory("sentinel-log-executor", true));
```

Comment:
You may need to refine the thread name so that we could distinguish the actual type of the logging (e.g. to file or to console).

Reply:
has renamed